### ISSUE TITLE
Plot Pre-Trained Resnet = Show the Model Summary?

### ISSUE THREAD

**Issue by OliverMathias** [jordanott/DeepLearning#2]
Quick question about the HW, for 3.2 it asks us to "Plot the Model." Does that mean we should show the summary of the model? Or evaluate it and print the accuracy before training somehow?
![plotmod](https://user-images.githubusercontent.com/44181723/65824942-9b60f200-e225-11e9-9747-d03ae77732e0.JPG)

Thanks!

**Comment by jordanott** [comment]
To get a visual graph, in your notebook you could run: 
```python
from IPython.display import SVG
from keras.utils.vis_utils import model_to_dot

SVG(model_to_dot(model).create(prog='dot', format='svg'))
```
If this doesn't work for you (has issues on windows) the summary is acceptable
